### PR TITLE
feature/fix-modal-issue

### PR DESCRIPTION
**Pull request by Humnaa**
Trello Link : https://trello.com/c/gR4cj2Ck/51-fix-modal-issue

### CODE REVIEW

File: src/containers/UserProfile/AboutUser.js [hunk 1]
```diff
@@ -11,25 +11,27 @@ class AboutUser extends Component {
     super(props);
     this.state = { about: this.props.about };
     this.validator = new SimpleReactValidator();
-    this.handleUserInput = this.handleUserInput.bind(this);
     this.notificationDOMRef = React.createRef();
-    this.submitForm = this.submitForm.bind(this);
-    this.onCancel = this.onCancel.bind(this);
-    this.modalRef = React.createRef();
+    this.backdropRef = React.createRef();
```
Comment (ayousaf11) [inline src/containers/UserProfile/AboutUser.js]: @Humnaa I cannot see any usage of it

File: src/containers/UserProfile/AboutUser.js [hunk 2]
```diff
@@ -11,25 +11,27 @@ class AboutUser extends Component {
...
+  };
 
-  onCancel() {
+  onCancel = () => {
     this.formRef.reset();
-  }
+  };
+
+  removeBackdrop = () => {
+    var parent = document.getElementsByClassName("modal-backdrop fade show");
```
Comment (ayousaf11) [inline src/containers/UserProfile/AboutUser.js]: @Humnaa rename `parent` to `element`

Comment (ayousaf11) [reply src/containers/UserProfile/AboutUser.js]: @Humnaa also add a check here i.e. `if(element && element.length)`

File: src/containers/UserProfile/EditIntro.js [hunk 3]
```diff
@@ -15,25 +15,27 @@ class EditInto extends Component {
...
-  onCancel() {
+  onCancel = () => {
     this.formRef.reset();
-  }
+  };
 
-  submitForm(e) {
+  removeBackdrop = () => {
+    var parent = document.getElementsByClassName("modal-backdrop fade show");
+    parent[0].remove();
```
Comment (ayousaf11) [inline src/containers/UserProfile/EditIntro.js]: @Humnaa rename `parent` to `element` and add check before accessing using `if(element && element.length)`

File: src/containers/UserProfile/EditIntro.scss [hunk 4]
```diff
@@ -49,3 +50,8 @@
     }
   }
 }
+.backdrop {
```
Comment (ayousaf11) [inline src/containers/UserProfile/EditIntro.scss]: @Humnaa is it required? Why are we adding a custom style for backdrop?

**Review by ayousaf11 — COMMENTED** [review]
@Humnaa please check my feedback

File: src/containers/UserProfile/AboutUser.js (hunk 1 above)
Comment (Humnaa) [reply src/containers/UserProfile/AboutUser.js]: Sorry I forgot to delete

File: src/containers/UserProfile/AboutUser.js (hunk 2 above)
Comment (Humnaa) [reply src/containers/UserProfile/AboutUser.js]: Renamed parent to element
and implement condition

File: src/containers/UserProfile/EditIntro.js (hunk 3 above)
Comment (Humnaa) [reply src/containers/UserProfile/EditIntro.js]: done

File: src/containers/UserProfile/EditIntro.scss (hunk 4 above)
Comment (Humnaa) [reply src/containers/UserProfile/EditIntro.scss]: same reason.. forgot to remove..

**Review by ayousaf11 — APPROVED** [review]

**Pull request merged**